hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -873,6 +873,22 @@
   </description>
 </property>
 
+<property>
+  <name>dfs.datanode.reconcile.blocks.batch.size</name>
+  <value>1000</value>
+  <description>HDFS-14476 to enable running reconcile to checkAndUpdate with batch,
```

Comment:
No need for jira id in the description, and this doesn't enable/disable, it is just to specify the batch size.